neo4j/auth_tokens.go

```diff
@@ -32,7 +32,9 @@ const schemeBearer = "bearer"
 const keyPrincipal = "principal"
 const keyCredentials = "credentials"
 const keyRealm = "realm"
-const keyTicket = "ticket"
+
+// Deprecated: will be removed in 5.0. Use keyCredentials instead.
+const keyTicket = keyCredentials
```

Comment:
since this is unexported (starts with a lower case letter), this can be removed (https://github.com/neo4j/neo4j-go-driver/pull/271 takes care of the deletion)

Reply:
:brain: